src/test/java/in/projecteka/consentmanager/consent/ConsentRequestUserJourneyTest.java

```diff
@@ -210,11 +211,12 @@ public void shouldAcceptConsentRequest() {
         when(conceptValidator.validatePurpose("CAREMGT")).thenReturn(Mono.just(true));
         when(conceptValidator.validateHITypes(any())).thenReturn(Mono.just(true));
         when(repository.requestOf(anyString())).thenReturn(Mono.empty());
+
         // TODO: Two calls being made to CR to get token within one single request, have to make it single.
-        load(clientRegistryServer, "{}");
-        load(clientRegistryServer, "{}");
-        load(clientRegistryServer, "{}");
-        load(clientRegistryServer, "{}");
+        load(clientRegistryServer, session);
+        load(clientRegistryServer, session);
+        load(clientRegistryServer, session);
+        load(clientRegistryServer, session);
```

Comment:
You don't have any new tests to check whether implementation works always? :(

Reply:
I didn't write any new as this is already being covered in the existing one's

Reply:
How it's covering caching part :thinking: ?